airflow/sensors/base_sensor_operator.py

```diff
@@ -42,9 +42,9 @@ class BaseSensorOperator(BaseOperator, SkipMixin):
     :type soft_fail: bool
     :param poke_interval: Time in seconds that the job should wait in
         between each tries
-    :type poke_interval: int
+    :type poke_interval: float
```

Comment:
I think int is the correct type, isn't it @mik-laj ?

We should probably update https://github.com/apache/airflow/pull/7050/files#diff-81901d44cd480538f1f35b641b9ede0cR70

Reply:
For me it should be a float here. We use it in two places. In timedelta (seconds) and in sleep. The second case is obvious, but the second is not, so I looked at code. 
![Screenshot 2020-01-05 at 16 22 58](https://user-images.githubusercontent.com/12058428/71782186-ac18c500-2fd7-11ea-9b8a-932ceb8f5b39.png)